code/datums/character_flaw/_character_flaw.dm

```diff
@@ -758,3 +759,30 @@ GLOBAL_LIST_INIT(character_flaws, list(
 						to_chat(H, span_warning("The weight of your equipment aggravates your chronic back pain!"))
 					BP.lingering_pain += pain_amount
 					break
+
+/datum/charflaw/witless_pixie
+	name = "Witless Pixie"
+	desc = "By some cruel twist of fate, you have been born a dainty-minded, dim-witted klutz. Yours is a life of constant misdirection, confusion and general incompetence. \
+	\nIt is no small blessing your dazzling looks make up for this, sometimes. \n\nSometimes... they do you no favours at all."
+
+/datum/charflaw/witless_pixie/on_mob_creation(mob/user)
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/H = user
+
+	H.change_stat(STATKEY_INT, rand(-2, -5)) //this would probably make the average manorc a vegetable
+
+/datum/charflaw/witless_pixie/after_spawn(mob/user)
+	if(!ishuman(user))
+		return
+
+	//solves edgecases with inbred princes and eoran hand-holders. Yes, you can be an ugly Eoran templar. You are not safe.
+	REMOVE_TRAIT(user, TRAIT_BEAUTIFUL, TRAIT_GENERIC)
+	REMOVE_TRAIT(user, TRAIT_UGLY, TRAIT_GENERIC)
+
+	if(prob(50))
+		ADD_TRAIT(user, TRAIT_BEAUTIFUL, TRAIT_GENERIC)
+
+	else if(prob(30))
+		ADD_TRAIT(user, TRAIT_UGLY, TRAIT_GENERIC)
+		return
```

Comment:
```suggestion
	if(prob(50))
		ADD_TRAIT(user, TRAIT_BEAUTIFUL, TRAIT_GENERIC)
	else if(prob(30))
		ADD_TRAIT(user, TRAIT_UGLY, TRAIT_GENERIC)
```
Also this is a 15% chance for ugly, 50% for beautiful and a 35% chance for nothing